### ISSUE TITLE
Deletion: Assume Y on Enter

### ISSUE THREAD

**Issue by alivefromupstairs** [gokcehan/lf#285]
When deleting a file and getting prompted to make sure

```
delete /home/me/file [y/N]?
```
Similar to situations like Arch's `pacman`: `:: Proceed with installation? [Y/n]`, the `y` is assumed on `<enter>`

Can we have that? I have delete mapped to `x` and to travel from `x` to `y` each time is a bit cumbersome. I am aware of the way `lf` tries to protect novice users from deleting important files, so, this could be optional and disabled by default.

**Comment by shabahengam** [comment]
no.delete command is a very important command and should be used with caution.

**Comment by alivefromupstairs** [comment]
I very much recognize that. perhaps, behind an option. one can send stuff to trash.
Maybe `y` to delete permanently, `n` to cancel deletion, and `enter` to send to trash (`/tmp`?). in this case, `enter` would be safer.

**Comment by gokcehan** [comment]
@alivefromupstairs Can you use a custom delete command for this purpose?

**Comment by alivefromupstairs** [comment]
I would really prefer not to, and if you decided to stick with this solution, I'll just do the same and curse every time I have to reach for `y`. also, @shabahengam I don't think `enter` will introduce any safety issues because `y` already is one click remove. you're just swapping a key for another.

**Comment by alivefromupstairs** [comment]
@gokcehan is there a place where I can find default commands definitions? so, if I ever decided to modify one, I don't have to start from scratch? I still think `lf` should have great sane defaults, though, that just work™.

**Comment by shabahengam** [comment]
if you use tmux sometimes you don't know which pane is active ([254](https://github.com/gokcehan/lf/issues/254))  and if you press `enter` in wrong pane (e.g. going to a directory) bad things will happen.

**Comment by alivefromupstairs** [comment]
That could be said for a tiling window manager. sometimes I have stuff in secondary selection and paste it with middle click to `nvim` supposedly, but `lf` explodes with all sorts of commands that I might make an issue about `lf` only accepting secondary selection on `cmd`. 

Ofc, in a window manager, the issue can be solved with a compositor. The issue, in your case is on `tmux`.

If you delete with `:delete` chance that you `enter` into `lf` is solid, unlike my WM who shifts focus on mouse hover which is easy to trigger (disabling that screws with floating windows).

**Comment by gokcehan** [comment]
@smhmd How do you decide what is sane and what is not? It seems to me that you're just pushing your own preferences as sane.

**Comment by smhmd** [comment]
It might seem that way, but if there is willingness to have better defaults, discussions such as this and other ones are the way to decide. Other factors that come into play:
- **Author preference:** Self explanatory. Completely valid.

- **Expectation:** Imagine a website that changes how `ctrl+c` behaves.

- **Familiarity:** People associate `lf` with `ranger` and `ranger` with TUIs. To ease migration, defaults could be taken directly from more established software unless for a reason or there is a coherent system behind the new defaults. (e.g., opening command line with `:`; vim keybindings.)

- **User input:** Our feedback. 

- **Minimal Configuration:** if most `lfrc` files on GitHub unmap a certain mapping or define a specific command, I think that is a good candidate to be a default, especially if the implementation is less than perfect. (e.g., `:rename`; `:open`).

- **User protection:**
	- `set ifs "\n"`. (Even though the default was the unexpected behavior for me.)
	- Not setting a key for `:delete`.


When it comes to keybindings and API, you can't just take people's opinion and run with it, of course. There has to be cohesion. However, implementation of core commands and actual interface (command names, UI and visual clues) are all up for debate until they stabilize.

Also, an actual default is not the only way to do it, it could be an option. E.g. closing `lf` closes programs opened with it? `$ echo set open async > /home/me/.config/lf/lfrc`.

**Comment by smhmd** [comment]
A suggestion to further protect from accidental deletion: items that are going to be deleted can have a reddish background for the duration of `delete 5 items [y/N]?` is displayed. Some visual clue is better than nothing changing.

**Comment by pc00per** [comment]
@smhmd Yah. Deletion in status bar should show red font as supposed to generic white.
Btw, even my 60% keyboard has physical delete button. If you have too, just use that.
`map <delete> delete` then `y`.
